test/remote/gateways/remote_litle_test.rb

```diff
@@ -132,9 +132,15 @@ def test_successful_authorization_with_echeck
     assert_equal 'Approved', response.message
   end
 
-  def test_avs_and_cvv_result
+  def test_avs_result
+    @credit_card1.number = 4100401234567000
     assert response = @gateway.authorize(10010, @credit_card1, @options)
     assert_equal 'X', response.avs_result['code']
+  end
+
+  def test_cvv_result
```

Comment:
💬  Comment:

In the same line of being clear on the test case, what is the meaning of that "M" ?, what are we checking ?

Reply:
> ❓ just personal curiosity question, Why separate this function, if we can use the same response?

It have been separated because of the sandbox behavior, it doesn't response with both replies simultaneously
<img width="891" alt="Screen Shot 2022-03-07 at 4 05 04 PM" src="https://user-images.githubusercontent.com/10518225/157117730-0e2bbdb1-dde2-4833-82e4-4dd034b59638.png">

Reply:
> 💬 Comment:
> 
> In the same line of being clear on the test case, what is the meaning of that "M" ?, what are we checking ?

That code is response for a "Match" in the card validation according to docs in:
http://support.worldpay.com/support/CNP-API/content/cardvalrespcodes.htm?Highlight=card%20validation